### ISSUE TITLE
Some Question about EHdn..

### ISSUE THREAD

**Issue by a7420174** [Illumina/ExpansionHunterDenovo#46]
Hello!

To detect ASD-related tandem repeat loci in my data, I'm refering to Trost et al. (2020). While I run an outlier analysis in EHdn, I have some questions.

1. I knew that depth-normalized counts are calculated as the raw count * 40 / the average read depth of the sample and the normalized counts are used to filter tandem repeat regions in Trost et al. (counts ≥ 2). I could understand the normalization of the counts by dividing the avg counts because of comparison between samples, but I had little understanding of the numerator. Is there any reason for setting a specific value (40)?

2. I got the counts of anchored IIRs in our cohort and an another cohort(1KGP) from EHdn. After filtering out some tandem repeats where too many samples have no count, I ran PCA of the count of anchored IIRs to check any ancestry or batch effect.

![image](https://user-images.githubusercontent.com/70624258/137059233-8e2c7d94-4e33-4615-9389-a189311d3538.png)

Here is the PCA result. It seems that they are split into our cohort(ASD; Korean) and 1KG samples. I checked the versions of gatk, bwa-mem, etc., but they were similar. Is there anything else to consider? Or maybe the count of anchored IIRs is not suitable for PCA?

3. I viewed tandem repeats manually from IGV to filter out false positives in the case of rare tandem repeats. Strangely, I found that the raw count of some anchored IIRs in EHdn profile seems different from the number of the reads in IGV.

![igv_ACTCCCACTGCCTCCCT_2](https://user-images.githubusercontent.com/70624258/137059461-17053216-06c1-4525-8c2f-17c88586bcba.png)

For example, this image is an IGV screenshot showing one anchored IIR checked.

```
contig  start   end     motif   num_anc_irrs    norm_num_anc_irrs       het_str_size
chr4    143052786       143052787       ACTCCCACTGCCTCCCT       2       1.53    9
```

And here is the EHdn result corresponding to the read above. I think two reads should appear in IGV. I wonder if I misread the result.

Thanks in advance.

**Comment by egor-dolzhenko** [comment]
Thanks for the questions Jaehyun!

1. No reason to use 40 other than convenience. We are essentially normalizing depth to 40x, which is pretty common for WGS data (if the actual depth is 40x than all IRR counts are unchanged after normalization.)

2. Hmm.. Do you know what sequencing instrument / library prep was used to generate your data? One way to check if your results are due to batch effects is to perform a complementary analysis by running (regular) [ExpansionHunter](https://github.com/Illumina/ExpansionHunter) on a [genome-wide repeat catalog](https://github.com/Illumina/RepeatCatalogs) that we just released and then run your PCA analysis on the resulting repeat genotypes.

3. Unfortunately IGV is not always suitable for confirming EHdn results because IRRs often misalign or don't get aligned at all. We have developed a dedicated read visualization tool for (regular) EH called [REViewer](https://github.com/Illumina/REViewer). In the future, we will try to extend support to EHdn as well.

Best wishes,
Egor

**Comment by a7420174** [comment]
Hi,

I have one more question about the tandem repeats not matched with TRF coordinates. Here is a plot for the distribution of motif length.

![](http://163.152.180.158:8787/graphics/7868413a-3106-49dd-99d3-25dbc3f3c644.png)

There are many VNTRs(longer than 7bp) not matched with TRF coordinates, and I understood that's possible because of the diversity of VNTR sequence composition. But the thing is the repeats of which motif length is 6bp. The percentage of AACCCT is ,surprisingly, 92% in 6bp motifs. But the percentage of AACCCT is not that high in TRF. I knew that these repeats consists of telomeres. I thought maybe the repeats in EHdn results are false positives. What do you think about it?

Again, thank you for taking the time to help.
 JaeHyun

**Comment by egor-dolzhenko** [comment]
Hi JaeHyun,

Sorry for the slow replies. This looks like an interesting observation that would be useful to follow up on.

We are currently working on a tool for automated assessment of EHdn results. I think this tool will be the perfect way to assess your results. Would you be interested in testing out the prototype of this tool? If yes, an early prototype should be ready by late November / early December.

Best wishes,
Egor

**Comment by a7420174** [comment]
Yes. It would be great if I take part in. I look forward to testing your tool!

Thanks,
JaeHyun
